src/assets/_project/_blocks/utils/qg-misc.js

```diff
@@ -25,21 +25,20 @@
   });
   // this function equals the height of the cards in a group, if it finds a class '.cards__equal-height'.
```

Comment:
This comment might need to be updated to '.qg-cards__equal-height'

Reply:
Thanks Ross. It is fixed in https://github.com/qld-gov-au/qg-web-template/pull/978